util/collate/collate.go

```diff
@@ -360,7 +360,8 @@ func ConvertAndGetBinCollation(collate string) Collator {
 func IsBinCollation(collate string) bool {
 	return collate == charset.CollationASCII || collate == charset.CollationLatin1 ||
 		collate == charset.CollationUTF8 || collate == charset.CollationUTF8MB4 ||
-		collate == charset.CollationBin
+		collate == charset.CollationBin || collate == "utf8mb4_0900_bin"
```

Comment:
Could we put "utf8mb4_0900_bin" to charset.go as a constant? Then we can call it like `charset.CollationUTF8MB4`.

Reply:
I think `CollationUTF8MB4` somehow represents the default collation for `utf8mb4` (it's `utf8mb4_bin` now). 

Maybe in the future PR I'll use `CollationUTF8MB40900Bin` to represent `utf8mb4_0900_bin` and replace every collation string literals, but not in this PR. I don't hope that big refractor hide the main modification made by this PR.

Reply:
Got it.